code/game/objects/items/handcuffs.dm

```diff
@@ -1,5 +1,6 @@
 /obj/item/restraints
 	breakouttime = 600
+	dye_color = DYE_PRISONER
```

Comment:
you sure this should be on base restraints? this will make beartraps, bolas, energy bolas turn into prisoner clothing

Reply:
Yeah, no harm in it I suppose. Washing Machines aren't using restraints' anyway, and I don't imagine many people wash their bear traps with their jeans.

Reply:
You, uh, do realize that washing machine dyeing doesn't actually changed the dyed object's typepath, right? It just makes it LOOK like a different item. So dyeing a unifom won't make it count for the bounty...

Reply:
Alternative provided in latest commit